src/item.h

```diff
@@ -495,6 +495,13 @@ class Item : virtual public Thing
 			return getIntAttr(ITEM_ATTRIBUTE_CORPSEOWNER);
 		}
 
+		void setRewardCorpse() {
+			setCorpseOwner(static_cast<uint32_t>(std::numeric_limits<int32_t>::max()));
```

Comment:
It's not clear here what owner uid value is supposed to mean. You should preferable put it into a typed contexpr(or const) with an appropriate name.